### DISCUSSION TITLE
Dateinput accepted dd/mm/yyyyy format immediately. Not after outside click calendar.

### DISCUSSION THREAD

**Discussion by Xotdog7** [mantinedev/mantine#7134] [Q&A]
Can you help with the Datepicker component. What I need to write so that during text input it accepts immediately the format dd/mm/yyyyyy, and not the moment when I click outside the calendar. onChange and onInput does not work for some reason

**Comment by rtivital** [comment]
I'm not aware of that format, it seems to be at least 10,000 years from now. Are you sure you need it?
You can use `dateParser` prop to parse these kinds of values – https://mantine.dev/dates/date-input/#date-parser
Additionally, make sure to include the custom parse plugin for dayjs like in this demo – https://mantine.dev/dates/date-input/#with-time

**Reply by Xotdog7** [reply]
I'm trying to make this component like here
https://mui.com/x/react-date-pickers/date-picker/#basic-usage
Dateparser does not work the way I would like it to work

**Reply by rtivital** [reply]
What in date parser does not work?
